BusinessLayer/SongPlayer/SongPlayer.cpp

```diff
@@ -4,11 +4,13 @@ namespace
 {
     const int MS_TO_MINUTES = 60000;
     const double MS_TO_SECONDS = 1000.0;
-    const int MIN_LIGHT = 40;
+    const int MIN_VALUE = 40;
```

Comment:
Whats this the min-value of?

Reply:
its literally the value component of hue-saturation-value

Reply:
Is this being used anywhere? A search through the project shows just this line.

Reply:
That goes for a couple of these variables, PAGE_STEP_INCREMENTS, and SKIP_PIXELS.